src/components/furniture/Shelf.vue

```diff
@@ -45,21 +45,28 @@ export default {
     return {};
   },
   computed: {
-    ...mapGetters(['totalWidth', 'shelfs']),
+    ...mapGetters(['totalWidth', 'shelfs', 'colorFurniture', 'colors']),
     cssStyle() {
       return {
         width: this.totalWidth * 3 + 'px',
         height: this.shelf.height * 3 + 'px',
-        borderTop: this.shelf.id === 0 ? '2px solid rgb(117, 62, 14)' : '',
+        borderTop: this.shelf.id === 0 ? '2px solid ' + this.colorBorder : '',
+        borderLeft: '2px solid ' + this.colorBorder,
+        borderRight: '2px solid ' + this.colorBorder,
+        borderBottom: '2px solid ' + this.colorBorder,
       };
     },
+    colorBorder() {
+      return this.colorFurniture.spot.includes('chants') && this.addColor
```

Comment:
you could extract `this.colorFurniture.spot.includes('chants')` to be a separate computed value